Backport of:
aad9729347e0a552f831622a587f41b196a843de
and
4da6751502111c52813cf23a40110f83aae501b9

which allows linking with an older efivar

diff --git a/plugins/dell/Makefile.am b/plugins/dell/Makefile.am
index 85c5269..47aa6ad 100644
--- a/plugins/dell/Makefile.am
+++ b/plugins/dell/Makefile.am
@@ -73,6 +73,7 @@ fu_dell_tool_SOURCES =					\
 fu_dell_tool_LDADD =					\
 	$(GLIB_LIBS)					\
 	$(FWUPD_LIBS)					\
+	$(EFIVAR_LIBS)					\
 	$(LIBSMBIOS_LIBS)
 
 fu_dell_tool_LDFLAGS =					\
diff --git a/plugins/synapticsmst/Makefile.am b/plugins/synapticsmst/Makefile.am
index d1577e7..a90337a 100644
--- a/plugins/synapticsmst/Makefile.am
+++ b/plugins/synapticsmst/Makefile.am
@@ -47,6 +47,7 @@ synapticsmst_tool_SOURCES =					\
 
 synapticsmst_tool_LDADD =					\
 	$(LIBSMBIOS_LIBS)					\
+	$(EFIVAR_LIBS)						\
 	$(APPSTREAM_GLIB_LIBS)					\
 	$(GLIB_LIBS)
 
@@ -71,6 +72,7 @@ fu_self_test_SOURCES =						\
 fu_self_test_LDADD =						\
 	$(FWUPD_LIBS)						\
 	$(GUSB_LIBS)						\
+	$(EFIVAR_LIBS)						\
 	$(LIBSMBIOS_LIBS)					\
 	$(APPSTREAM_GLIB_LIBS)					\
 	$(SQLITE_LIBS)						\
